File: pm_core_1/src/core/domain/entity/issue_type_scheme_item_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type IssueTypeSchemeItemEntity struct {
	BaseEntity
	TenantID    int64 `json:"tenantId"`
	SchemeID    int64 `json:"schemeId"`
	IssueTypeID int64 `json:"issueTypeId"`
	Sequence    int   `json:"sequence"`
}

File: pm_core/src/main/java/serp/project/pmcore/domain/issyetype/entity/IssueTypeSchemeItemEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.domain.issyetype.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;
import serp.project.pmcore.domain.shared.entity.BaseEntity;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class IssueTypeSchemeItemEntity extends BaseEntity {
    private Long tenantId;
    private Long schemeId;
    private Long issueTypeId;
    private Integer sequence;
}
